overrides.go

```diff
@@ -0,0 +1,72 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Contributor:
+// - Brandon Myers bmyers@mozilla.com
+
+package main
+
+import (
+  "fmt"
+  "os"
+  "bufio"
+  "strings"
+  "strconv"
+)
+
+type Override struct {
+  CIDR string
+  City string
+  Country string
+  Latitude float64
+  Longitude float64
+}
+
+
+func readOverrides(path string) (overrides []Override){
```

Comment:
Just confirming this but I'm guessing the intent here is to try to load the overrides file and just continue on if we can't find it or there is a syntax error in it; should we return an error from this function instead in those cases? Or if there are syntax errors in certain lines in the file indicate those while we are parsing it?

Reply:
Yeah, the idea was that if the overrides file wasn't there, or was malformed, we could still continue on. If we want to keep that assumption, it might make sense to return an error from this function like you recommended, and have the caller of the function be the one to decide what it should do with the error (in this case, just ignore it, and set the overrides array to [])

Reply:
I ended up switching some of the original logic around in https://github.com/ameihm0912/geomodel/pull/10/commits/7924535a214c3750e5ab7e2b4d0e4058e63dc2c1  What I actually implemented was if the overrides config file isn't present, we'll exit out (compared to defaulting to an empty array and continuing on). The reasoning for this is, folks can use the stub overrides.conf file if they don't want to specify any overrides.

I also moved this decision logic from the function that parses the override config, into main.go. If the override config parsing function encounters any errors, it'll bubble them up to the caller.
 Thoughts?